api/src/test/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminServiceTest.java

```diff
@@ -165,7 +168,6 @@ public class WorkspaceAdminServiceTest {
     UserMapper.class,
     UserService.class,
     WorkspaceAuthService.class,
-    WorkspaceService.class
```

Comment:
this is now the individually accessible `@MockBean private WorkspaceService mockWorkspaceService;`